datasets/kws20.py

```diff
@@ -38,6 +38,7 @@
 
 import librosa
```

Comment:
you may want to remove import librosa if not needed anymore

Reply:
librosa is still needed for loading the audio